rj_utils/include/rj_utils/logging.hpp

```diff
@@ -0,0 +1,189 @@
+#pragma once
+
+/**
+ * \file logging.hpp
+ * \brief This file defines macros that wrap the ones in rclcpp/logging.hpp
+ * but with a NOLINT at the end, so that we don't trip up clang_tidy every time
+ * we make a logging call. This also defines EZ prefixed log calls that
+ * automatically use get_logger() to get the current logger, so that logging
+ * calls can be shorter.
+ */
+
+#include <rclcpp/logging.hpp>
+
+// Disable formatting on this entire section so NOLINT works properly
+// clang-format off
+//======================================================================
+// DEBUG
+//======================================================================
+#define RJ_DEBUG(logger, ...) RCLCPP_DEBUG(logger, __VA_ARGS__)  // NOLINT
+#define RJ_DEBUG_ONCE(logger, ...) RCLCPP_DEBUG_ONCE(logger, __VA_ARGS__)  // NOLINT
+#define RJ_DEBUG_EXPRESSION(logger, expression, ...) RCLCPP_DEBUG_EXPRESSION(logger, expression, __VA_ARGS__)  // NOLINT
+#define RJ_DEBUG_FUNCTION(logger, function, ...) RCLCPP_DEBUG_FUNCTION(logger, function, __VA_ARGS__)  // NOLINT
+#define RJ_DEBUG_SKIPFIRST(logger, ...) RCLCPP_DEBUG_SKIPFIRST(logger, __VA_ARGS__)  // NOLINT
+#define RJ_DEBUG_THROTTLE(logger, clock, duration, ...) RCLCPP_DEBUG_THROTTLE(logger, clock, duration, __VA_ARGS__)  // NOLINT
+#define RJ_DEBUG_SKIPFIRST_THROTTLE(logger, clock, duration, ...) RCLCPP_DEBUG_SKIPFIRST_THROTTLE(logger, clock, duration, __VA_ARGS__)  // NOLINT
+
+#define RJ_DEBUG_STREAM(logger, stream_arg) RCLCPP_DEBUG_STREAM(logger, stream_arg)  // NOLINT
+#define RJ_DEBUG_STREAM_ONCE(logger, stream_arg) RCLCPP_DEBUG_STREAM_ONCE(logger, stream_arg)  // NOLINT
+#define RJ_DEBUG_STREAM_EXPRESSION(logger, expression, stream_arg) RCLCPP_DEBUG_STREAM_EXPRESSION(logger, expression, stream_arg)  // NOLINT
+#define RJ_DEBUG_STREAM_FUNCTION(logger, function, stream_arg) RCLCPP_DEBUG_STREAM_FUNCTION(logger, function, stream_arg)  // NOLINT
+#define RJ_DEBUG_STREAM_SKIPFIRST(logger, stream_arg) RCLCPP_DEBUG_STREAM_SKIPFIRST(logger, stream_arg)  // NOLINT
+#define RJ_DEBUG_STREAM_THROTTLE(logger, clock, duration, stream_arg) RCLCPP_DEBUG_STREAM_THROTTLE(logger, clock, duration, stream_arg)  // NOLINT
+#define RJ_DEBUG_STREAM_SKIPFIRST_THROTTLE(logger, clock, duration, stream_arg) RCLCPP_DEBUG_STREAM_SKIPFIRST_THROTTLE(logger, clock, duration, stream_arg)  // NOLINT
+
+#define EZ_DEBUG(...) RCLCPP_DEBUG(get_logger(), __VA_ARGS__)  // NOLINT
```

Comment:
Do we ever anticipate using the `RJ_*` macros from above? I think we'll almost always be using the default logger. Why not make the default ones `RJ_*` and name the custom logger ones something else?

Reply:
Yeah, I think we'll usually be using the default logger. There might be use cases for using different loggers for like when a node is multithreaded and you want to filter out logs by thread or something.

Reply:
Okay, we'll cross that bridge when we get to it I guess.